### PR TITLE
Graph API demo with Swagger

### PR DESCRIPTION
**Pull request by CiroGamboa**
There is a Swagger API that provides a set of Flask enpoints that handle the text extraction and construction of the graph on Neo4j. The Python code connects to an external instance of Neo4j, which can be set using env variables. CRUD operations on the nodes are possible.

### CODE REVIEW

File: climate_scanner/entity_network/graph_demo/app.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+		'in': 'header',
+		'name': 'Authorization'
+	},
+}
+
+api = Api(app=app, authorizations=authorizations)
+
+# Load in key from environment vars typically
+# SWAGGER_KEY = os.environ.get('SWAGGER_KEY')
+SWAGGER_KEY = '1234'
```
Comment (Eliotfrazier) [inline climate_scanner/entity_network/graph_demo/app.py:31]: Same here...

**Review by Eliotfrazier — APPROVED** [review]
Looks good overall, a couple of minor comments..

File: climate_scanner/entity_network/graph_demo/entity_extraction.py [hunk 2]
```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
```
[the remark this answers is not in the document]
Comment (CiroGamboa) [reply climate_scanner/entity_network/graph_demo/entity_extraction.py:1]: The files that are on the demo folder contain all the work I did. The ones that are outside, are just as I received them, thats work from Skander I think, so I wanted to keep them unmodified for now, till we clarify some requirements

File: climate_scanner/entity_network/entity_extraction.py [hunk 3]
```diff
@@ -2,11 +2,14 @@
...
 import requests
 import os
-import spacy 
+import spacy
 nlp = spacy.load('en_core_web_lg')
 
 # Set via environment variables
-os.getenv('user_key')
+#os.getenv('user_key')
+user_key = "beejatbmicwzpxudjxxslszpidsjep"
```
[the remark this answers is not in the document]
Comment (CiroGamboa) [reply climate_scanner/entity_network/entity_extraction.py:10]: Oh, I did it in the file that's inside the demo folder, I forgot to do it here, same with the swagger key. I'll do it

**Comment by Eliotfrazier** [comment]
Closing for now.

**Pull request closed without merging**